build.gradle.kts

```diff
@@ -10,7 +10,7 @@ plugins {
 }
 
 group = "uk.nhs.hee.tis.trainee"
-version = "0.29.2"
+version = "0.30.2"
```

Comment:
30.2 ? Future-proofing! :>

Reply:
It was three separate increments originally, just ended up PRed together 🙈